eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml

```diff
@@ -38,39 +38,48 @@ steps:
       archType: ${{ parameters.archType }}
       buildConfig: ${{ parameters.buildConfig }}
       testBuildArgs: ${{ parameters.testBuildArgs }}
-
   # Build a Mono LLVM AOT cross-compiler for non-amd64 targets (in this case, just arm64)
-  - ${{ if and(eq(parameters.runtimeFlavor, 'mono'), or(eq(parameters.runtimeVariant, 'llvmaot'), eq(parameters.runtimeVariant, 'llvmfullaot'))) }}:
+  - ${{ if and(eq(parameters.runtimeFlavor, 'mono'), in(parameters.runtimeVariant, 'llvmaot', 'llvmfullaot', 'minifullaot')) }}:
     - ${{ if eq(parameters.archType, 'arm64') }}:
-      - script: ./build.sh
-                -subset mono
-                -c ${{ parameters.buildConfig }}
-                -arch ${{ parameters.archType }}
-                /p:BuildMonoAotCrossCompiler=true
-                /p:BuildMonoAotCrossCompilerOnly=true
-                /p:MonoLibClang="/usr/local/lib/libclang.so.16"
-                /p:MonoAOTEnableLLVM=true
-                /p:CrossBuild=true
-        displayName: "Build Mono LLVM AOT cross compiler"
+      - ${{ if eq(parameters.runtimeVariant, 'minifullaot') }}:
+        - script: ./build.sh
+                  -subset mono
+                  -c ${{ parameters.buildConfig }}
+                  -arch ${{ parameters.archType }}
+                  /p:BuildMonoAotCrossCompiler=true
+                  /p:BuildMonoAotCrossCompilerOnly=true
+                  /p:CrossBuild=true
+          displayName: "Build Mono Mini AOT cross compiler"
+      - ${{ else }}:
+        - script: ./build.sh
+                  -subset mono
+                  -c ${{ parameters.buildConfig }}
+                  -arch ${{ parameters.archType }}
+                  /p:BuildMonoAotCrossCompiler=true
+                  /p:BuildMonoAotCrossCompilerOnly=true
+                  /p:MonoLibClang="/usr/local/lib/libclang.so.16"
+                  /p:MonoAOTEnableLLVM=true
```

Comment:
I'm not sure if we shouldn't also pass `/p:MonoEnableLLVM=true`. For Perf AOT runs we use `/p:MonoAOTEnableLLVM=true /p:MonoEnableLLVM=true /p:BuildMonoAOTCrossCompiler=true ...` for both x64 and arm64.

Reply:
Good point. Based on the source code, the output directory should be different for the LLVM optimizer and the linker when `MonoAOTEnableLLVM` vs `MonoEnableLLVM` is used. I suggest to address it in the LLVM-related PR.